core/src/main/java/brooklyn/event/feed/windows/WindowsPerformanceCounterFeed.java

```diff
@@ -251,10 +258,71 @@ public T call() throws Exception {
         }
     }
 
+    private static class SendPerfCountersToSensors implements PollHandler<WinRmToolResponse> {
+
+        private final EntityLocal entity;
+        private final List<WindowsPerformanceCounterPollConfig<?>> polls;
+
+        public SendPerfCountersToSensors(EntityLocal entity, List<WindowsPerformanceCounterPollConfig<?>> polls) {
+            this.entity = entity;
+            this.polls = polls;
+        }
+
+        @Override
+        public boolean checkSuccess(WinRmToolResponse val) {
+            if (val.getStatusCode() != 0) return false;
+            String stderr = val.getStdErr();
+            if (stderr == null || stderr.length() != 0) return false;
```

Comment:
Can we not just rely on the statusCode?
Worth a TODO to explain why treating presence of stderr as a failure.